charts/deepgram-self-hosted/CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/),
 ### Added
 
 - Automatic model management on AWS now supports model removal. See the `engine.modelManager.models.remove` section in the `values.yaml` file for details.
+- Container orchestrator environment variable added to improve support.
```

Comment:
Worth calling out the 3 supported values, since it's a small finite set and we could potentially add/change it in the future?

Reply:
It's not necessarily intended to be change or configured - I don't think it needs to be called out.

Also, the cardinality will increase, as the Helm value will change for each version (but the prefix will stay the same).